packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -169,6 +170,7 @@ export async function summarizeNow(summarizer: ISummarizer, reason: string = "en
 
 	const submitResult = await timeoutAwait(result.summarySubmitted);
 	if (!submitResult.success) {
+		submitResult.error.data = submitResult.data;
```

Comment:
This isn't great. @markfields Any ideas on how to make this better?

Reply:
Eh, no I can't think of anything better.  The types do seem a bit redundant / confusing as this data flows around, but nothing obvious jumping out for how to untangle it here.

I don't mind weird stuff like this as much in tests because you get runtime validation via the test itself, so bending the type system is less concerning.

Reply:
I agree. The types and properties don't transition natually in the whole summarization flow. Maybe we can clean that up as part of the summarize observability / telemetry work.